### PR TITLE
[Chat] Add isOpen to image gallery's API

### PR DESCRIPTION
**Pull request by Leah-Xia-Microsoft**
# What

Add isOpen to image gallery's API

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/communication-react/review/beta/communication-react.api.md [hunk 1]
```diff
@@ -2764,6 +2764,7 @@ export interface ImageGalleryImageProps {
 // @beta
 export interface ImageGalleryProps {
     images: Array<ImageGalleryImageProps>;
+    isOpen?: boolean;
```
Comment (JoshuaLai) [inline packages/communication-react/review/beta/communication-react.api.md:2767]: Should we make this required instead? I'm not sure what value having this as optional would be

Comment (jimchou-dev) [inline packages/communication-react/review/beta/communication-react.api.md:2767]: `isOpen` sounds a bit odd to me, maybe `isVisible` or `isShown` or `isActive` or `isOpened`?

Comment (Leah-Xia-Microsoft) [reply packages/communication-react/review/beta/communication-react.api.md:2767]: > `isOpen` sounds a bit odd to me, maybe `isVisible` or `isShown` or `isActive` or `isOpened`?

I think we briefly talked about this in the last internal review and we tend to agree to follow the naming of the FluentUI Modal component which is isOpen. @JoshuaLai Do you remember whether my statement is correct?

Comment (JoshuaLai) [reply packages/communication-react/review/beta/communication-react.api.md]: Yes. If isOpen is what FluentUI uses lets try to match their naming

**Review by JoshuaLai — APPROVED** [review]

**Review by jimchou-dev — APPROVED** [review]

**Pull request merged**